### ISSUE TITLE
display order for DIAMM news (flat pages)

### ISSUE THREAD

**Issue by juliacmcf** [diamm/diamm#249]
Would it be possibly to have the most recent news item appear at the top of the list on the front page? Incidentally the newsfeed on the front page doesn't appear to have picked up the two new items that I put up today.

**Comment by ahankinson** [comment]
Did you add then through the CMS admin site? It's the one with the bird on it ("wagtail"). It won't show up if you just add the page in the database admin site.

**Comment by juliacmcf** [comment]
Yes, did it in Wagtail. The pages are showing up, but with the oldest stuff first. I don’t know how to reverse the order.

**Comment by ahankinson** [comment]
OK, I'll look into it.

**Comment by juliacmcf** [comment]
In fact it looks as if the new news pages are not showing up on the front page at all. May be something I have to do in Wagtail, but I've just clicked on 'show in menus' in the promote thing and they're still not showing, and I've checked I'm not just looking at a cached page.

**Comment by ahankinson** [comment]
Ah. You added a 'Content Page' not a 'News Page.' When you click 'Add a child page' it will give you the option of choosing several types. If you want "News" you need to add it as a News page.

**Comment by ahankinson** [comment]
PS: Not a "News Index" page either...

**Comment by juliacmcf** [comment]
Rats!

**Comment by juliacmcf** [comment]
They are coming up in the correct order too!!

Sorry for being a doofus.
